pkgs/applications/terminal-emulators/tabby/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchurl, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  name = "tabby";
+  version = "1.0.207";
+  src = fetchurl {
+    url = "https://github.com/Eugeny/tabby/releases/download/v${version}/tabby-${version}-linux-x64.deb";
+    hash = "sha256-PTTkL3+mYb7KM8fDUmgCuAF2lU88fYOstGWp/O5WZas=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    dpkg
+    makeWrapper
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg -x $src .
+
+    runHook postUnpack
+  '';
+
+  buildInputs = with pkgs; [
+    glib
+    glibc
+    nss
+    nspr
+    atk
+    cups
+    dbus
+    libdrm
+    gtk3
+    pango
+    cairo
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    libxkbcommon
+    mesa
+    expat
+    xorg.libxcb
+    alsaLib
+    libGL
+    libsecret
+    musl
+  ];
+
+  postInstall = ''
+    mkdir -p $out/{opt,bin}
+    cp -r opt $out
+    wrapProgram "$out/opt/Tabby/tabby" --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland --enable-features=WaylandWindowDecorations}}" --set LD_LIBRARY_PATH=${lib.makeLibraryPath buildInputs}
+    cp -r usr/* $out
+
+    cat > $out/bin/tabby << END
+    #!/bin/sh
+    '$out/opt/Tabby/tabby' --no-sandbox $@
```

Comment:
There should preferably be an explanation on what --no-sandbox does.

Reply:
Idk what it does, but that's what the official Tabby deb has